README.md

```diff
@@ -1 +1,42 @@
+* The majority of participating reviewers rejects the proposal → The proposal is rejected. The participating reviewers and the author will consider whether any part part of the proposal can be improved and/or incorporated into a new proposal
+Community contributors are encouraged to present proposals and to share their opinions on any Request For Enhancement (RFE). However, any approval throughout the workflow must be granted by the Narayana team and its optional members.
+
+\* The time window can be adjusted according to individual and team circumstances. However, the Narayana team is required to provide a different time window within 3 days from the date a new proposal is created.
+
+## 1. Proposal and technical document
+
+To ensure a clear connection between proposals and their corresponding tasks, a dedicated folder for each proposal will be created within this repository. The folder name will be the unique JIRA issue associated with the RFE. This JIRA issue should be opened before creating the proposal.
+
+In order to facilitate open discussions and transparent collaboration, a pull request for the technical document must be raised for each proposal. The discussion will thus evolve through reviews and feedback. When the proposal is given the go-ahead by the reviewers, the pull request will be merged.
+
+Once the technical document has been approved and merged, it cannot be modified. If any improvement or modification is needed, the new technical document will have to be reviewed and approved by all reviewers.
+
+To facilitate the navigation between the pull request and the related JIRA, a hyperlink to the JIRA should be added to the description of the pull request and, vice versa, a link to the pull request should be attached to the JIRA.
+
+## 2. Implementation Document (optional)
+
+An optional implementation-specific document can be submitted at this stage. This document should contain a clear roadmap to ensure a smooth transition from design to implementation. Although this step is optional, it might become necessary when there are disagreements among reviewers. Any time there is a disagreement on the implementation of the technical document, a pull request for an implementation-specific document must be raised and a discussion should be initiated. Once the Narayana team agrees on a specific implementation, the pull request should be merged and the RFE must be developed based on that document.
+
+Once the implementation-specific document has been approved and merged, it cannot be modified. If any improvement or modification is needed, the new implementation-specific document will have to be reviewed and approved by all reviewers.
+
+To facilitate the navigation between the pull request and the related JIRA, a hyperlink to the JIRA should be added to the description of the pull request and, vice versa, a link to the pull request should be attached to the JIRA.
```

Comment:
So we have two pull requests in play: the proposal and the implementation document?

Reply:
Yep, that is correct, however, they have different purposes. The technical document (i.e. the template in this PR) should provide an overall technical view of the proposal while the implementation document should provide a platform to discuss implementation details. Bear in mind that the latter is optional. In fact, it is up to the author of the PR to submit an implementation document to reach an agreement before starting the actual development of their RFE. In other words, the implementation document can be used to get the approval before starting coding. Once the approval of the implementation document is secured, the PR implementing the RFE (taken for granted that it implements the design discussed in the implementation document) will be automatically approved and merged (of course, issues like bugs, wordings, oversights, anti-patterns, etc. should still be addressed before merging the PR but the overall design of the PR should not be challenged).

Reply:
I think a format for the `Implementation Document` should be provided and linked

Reply:
This is especially important to understand how that can be used when `Any time there is a disagreement on the implementation of the technical document, a pull request for an implementation-specific document must be raised and a discussion should be initiated.`

Reply:
I don't believe we should be overly prescriptive when it comes to the implementation document. The only 'template' I can think of is that the author of the proposal should outline their approach for fulfilling each requirement specified in the technical document. However, I don't have many other ideas about how to guide the author in explaining how they intend to meet these requirements.

Reply:
Nevertheless, I will add a simple template for the implementation document to this PR so that other contributors can discuss and propose improvements.

Thanks for your point @tomjenkinson

Reply:
Thank you! I think in the paragraph two before this one there is some possible content for that document that is currently in this document. I will highlight it.

Reply:
For me, this topic is resolved. @mmusgrov as you opened it, please can I ask if you have anything against me resolving this conversation?